.github/workflows/pr-checks.yml

```
@@ -0,0 +1,260 @@
+
+  # PR Labels
+  label-pr:
+    name: Auto Label PR
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+      
+    steps:
+      - name: Label PR based on files
+        uses: actions/labeler@v4
```

actionlint (1.7.7)

the runner of "actions/labeler@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)